Client/mods/deathmatch/logic/lua/CLuaMain.cpp

```diff
@@ -143,20 +143,14 @@ void CLuaMain::InitVM()
     assert(!m_luaVM);
 
     // Create a new VM
-    m_luaVM = lua_open();
+    m_luaVM = luaL_newstate();
     m_pLuaManager->OnLuaMainOpenVM(this, m_luaVM);
 
     // Set the instruction count hook
     lua_sethook(m_luaVM, InstructionCountHook, LUA_MASKCOUNT, HOOK_INSTRUCTION_COUNT);
 
     // Load LUA libraries
-    luaopen_base(m_luaVM);
-    luaopen_math(m_luaVM);
-    luaopen_string(m_luaVM);
-    luaopen_table(m_luaVM);
-    luaopen_debug(m_luaVM);
-    luaopen_utf8(m_luaVM);
-    luaopen_os(m_luaVM);
+    luaL_openlibs(m_luaVM);
```

Comment:
Huge discussion about libs here #1972 and here https://github.com/multitheftauto/mtasa-blue/discussions/1782